### PR TITLE
Redirect github.io to RTD url

### PR DESCRIPTION
**Pull request by lgeorge-idm**
- [x] Added the necessary code, but the `rtd_url` value in docs/conf.py:184 needs to be changed to the real RTD URL.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -181,6 +181,7 @@
 html_static_path = ['_static']
 
 html_context = {
+    'rtd_url': 'https://institutefordiseasemodeling.github.io/synthpops/',
```
Comment (lgeorge-idm) [inline docs/conf.py:184]: @JSchripsema-IDM this is where you should put in the correct RTD url.

**Comment by lgeorge-idm** [comment]
@JSchripsema-IDM, I won't merge this until the SynthPops documentation on docs.idmod.org is made public.

**Review by JSchripsema-IDM — APPROVED** [review]
Synthpops docs have been made public at https://docs.idmod.org/projects/synthpops/en/latest/

**Comment by JSchripsema-IDM** [comment]
@lgeorge-idm It's not clear to me why the redirects don't appear to be working here.

**Comment by lgeorge-idm** [comment]
I'm rebuilding the gh-pages branch now. Not sure why this isn't working either.

**Pull request merged**